packages/apps/job-launcher/server/src/modules/cron-job/cron-job.service.ts

```diff
@@ -37,17 +37,17 @@ export class CronJobService {
   ) {}
 
   public async startCronJob(cronJobType: CronJobType): Promise<CronJobEntity> {
-    let cronJob = await this.cronJobRepository.findOneByType(cronJobType);
+    const cronJob = await this.cronJobRepository.findOneByType(cronJobType);
 
     if (!cronJob) {
-      cronJob = await this.cronJobRepository.createUnique(cronJobType);
+      const cronJobEntity = new CronJobEntity();
+      cronJobEntity.cronJobType = cronJobType;
+      return this.cronJobRepository.createUnique(cronJobEntity);
     } else {
```

Comment:
Usually you don't need `else` statement, because `if (!cronJob)` false, you will skip `if` block.